drivers/TSP01/driver.js

```diff
@@ -0,0 +1,265 @@
+'use strict';
+
+const path = require('path');
+const ZwaveDriver = require('homey-zwavedriver');
+
+// http://www.tkbhome.com/?cn-p-d-258.html
+// http://www.philio-tech.com/pdf/PSP01.pdf
+// https://www.intellihome.be/nl/z-wave-tkb-3-in-1-sensor-pir-light-temperature.html
+// TSP01
+
+module.exports = new ZwaveDriver( path.basename(__dirname), {
+	capabilities: {
+		'alarm_motion': {
+			'resetTimer'                : 'motion_inactivity_timer',	// Can reference to a device setting or a number.
+			'resetTimerCallback'        : function (capabilityRef, refNo, node) {
+				if (capabilityRef.refNo == refNo) {
+					module.exports._debug(`Resetting state for ${node.device_data.token}/alarm_motion to false` )
+					// Parse value
+					const value = false
+
+					// Update value in node state object
+					node.state['alarm_motion'] = value;
+
+					// Emit realtime event
+					module.exports.realtime(node.device_data, 'alarm_motion', value);
+				} else {
+					module.exports._debug(`Not resetting state for ${node.device_data.token}/alarm_motion, refNo ${refNo} != ${capabilityRef.refNo}`)
+				}
+			},
+			'command_basic_usage'       : 'motion_interact_class_basic',
+			'command_class'             : 'COMMAND_CLASS_SENSOR_BINARY',
+			'command_report'            : 'SENSOR_BINARY_REPORT',
+			'command_report_parser'     : function( report, node ){
+				// If we got a resolved timeout setting, then act on it
+				if (this._resetTimer) {
+					let refNo = Math.random().toString()
+					this.refNo = refNo;
+
+					module.exports._debug(`Got report for ${node.device_data.token}/alarm_motion, reset after ${this._resetTimer} milliseconds if refNo == ${refNo}`)
+					setTimeout(this.resetTimerCallback, this._resetTimer, this, refNo, node)
+				}
+				if (report['Sensor Type'] == 'Motion') {
+					// setTimeout(function() {node.}, 6000)
+					return report['Sensor Value (Raw)'].toString('hex') == 'ff';
+				}
+			}
+		},
+		'alarm_tamper': {
+			'command_class'             : 'COMMAND_CLASS_SENSOR_BINARY',
+			'command_report'            : 'SENSOR_BINARY_REPORT',
+			'command_report_parser'     : function( report ){
+				if (report['Sensor Type'] == 'Tamper') {
+					return report['Sensor Value (Raw)'].toString('hex') == 'ff';
+				}
+			},
+			'optional': true
+		},
+		'measure_luminance': {
+			'command_class'             : 'COMMAND_CLASS_SENSOR_MULTILEVEL',
+			'command_report'            : 'SENSOR_MULTILEVEL_REPORT',
+			'command_report_parser'     : function( report ){
+				if (report['Sensor Type'] == 'Luminance (version 1)') return parseInt(report['Sensor Value (Parsed)']) * 10;
+			},
+			'optional': true
+		},
+		'measure_temperature': {
+			'command_class'             : 'COMMAND_CLASS_SENSOR_MULTILEVEL',
+			'command_report'            : 'SENSOR_MULTILEVEL_REPORT',
+			'command_report_parser'     : function( report ){
+				if (report['Sensor Type'] == 'Temperature (version 1)') return (parseInt(report['Sensor Value (Parsed)']) - 32) * 5 / 9;
```

Comment:
Maybe then remove the setting at all (also the other one with the bit-wise settings), because it will be hard to setup anyway (because you have to manually enter the decimal value while doing bits and stuff...)

It just tried to be complete, though i doubt someone will change those settings...